encode_utils/profiles.py

```diff
@@ -82,9 +82,25 @@ def remove_duplicate_associations(associations):
         if val.startswith("/"):
             prefix = inflection.singularize(val.strip("/").split("/")[0])
             if prefix in Profile.PROFILES:
-                associations[i] = val.split("/")[-1]
+                associations[i] = val.strip("/").split("/")[-1]
     return list(set(associations))
 
+def remove_duplicate_objects(objects):
+    """
+    Checks for duplicates in array properties containing dictionary elements.
+
+    Args:
+        objects: `list`.
```

Comment:
There should be an empty line between the Args and Returns lists in the docstring to be consistent with the rest of the code